skimage/color/colorconv.py

```diff
@@ -753,7 +753,10 @@ def rgb2gray(rgb):
     rgb : array_like
         The image in RGB format, in a 3-D or 4-D array of shape
         ``(.., ..,[ ..,] 3)``, or in RGBA format with shape
-        ``(.., ..,[ ..,] 4)``.
+        ``(.., ..,[ ..,] 4)``
```

Comment:
We have to remove the mention to arrays of shape ``(.., ..,[ ..,] 4)`` since `_prepare_colorarray` doesn't accept RGBA images.